src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,46 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val genericGreetings = listOf("Hello world", "Salut tout le monde", "Hola, mundo")
+
+        val EnglishGreeting: HelloWorldGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Hello"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Hello $someone"
+            }
+        }
+
+        val FrenchGreeting: HelloWorldGreeting = object : HelloWorldGreeting {
+            override fun greet(): String {
+                return "Salut"
+            }
+
+            override fun greetSomeone(someone: String): String {
+                return "Salut $someone"
+            }
+        }
+
+        val SpanishGreeting: HelloWorldGreeting = object : HelloWorldGreeting {
```

Comment:
The variable name `SpanishGreeting` should start with a lowercase letter according to Kotlin naming conventions. Consider renaming it to `spanishGreeting`.